docs/hyn/5.2/tinker.md

```diff
@@ -5,9 +5,9 @@ icon: fal fa-screwdriver
 
 # Tenant Aware Tinker Command
 
-This tinker documentation will allow you to extend the tinker command and make it tenant aware.
+This documentation will allow you to extend the tinker command and make it tenant aware.
```

Comment:
What about `This custom tinker command will ..`. I think `This documentation` refers too much to the full documentation as a whole.

Reply:
If we do that, I would add a modifier on the second mention of the tinker command:

`This custom tinker command will allow you to extend the Laravel framework tinker command and make it tenant aware.`

Or go back to the original phrasing with the Laravel modifier: 
`This tinker documentation will allow you to extend the Laravel framework tinker command and make it tenant aware.`

Somewhere we need to distinguish what we're modifying so it doesn't sound like we're extending itself.

Side note: would this tinker functionality be best folded into the framework as a feature and then document the usage of it in this doc?

Reply:
> Side note: would this tinker functionality be best folded into the framework as a feature and then document the usage of it in this doc?

@bkintanar asked me the same. To be honest, the tinker command has been made optional in Laravel. That's the sole reason to keep it out of hyn too. Perhaps, seeing how much unnecessary documentation it requires, it should be offered in the package. I'll consider it.

How about this:

> This article describes how to enable tenant awareness in the native Laravel tinker command.

Reply:
Ah. Solid reasoning. Maybe maintain it as its own package which can be installed as another composer dependency, similar to how Laravel has accomplished it. That would make this documentation very simple and also keep it's optionality.

Doing it as a package would allow you to define Laravel's tinker as a dependency to overcome anyone who may not have it in their dependencies yet.

I like your final draft. I will push an updated commit.

Reply:
Updated "in the native" to "for the native" in the commit, fyi.